meteor-app/imports/ui/pages/home/index-header.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Home page.
+ */
+
+import React from 'react';
+import { FlowRouter } from 'meteor/ostrio:flow-router-extra';
+import TextMenu from '/imports/ui/components/textmenu';
+
+export function IndexHeader() {
+  const dataVar = [];
+  const itemsArr = [{ type: 'menuitem', label: 'SKOPE NSF Proposal' },
+                  { type: 'menuitem', label: 'SKOPE Prototype' },
+                  { type: 'menuitem', label: 'Stories' }];
+  dataVar.push({ type: 'menu', label: 'What is SKOPE?', items: itemsArr });
```

Comment:
What's the reason to push everything into the array instead of composing the array with these data in it in the first place?